source/patterns/@aws-solutions-constructs/aws-iot-sqs/lib/index.ts

```diff
@@ -0,0 +1,159 @@
+/**
+ *  Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+import { Queue, QueueProps, DeadLetterQueue } from '@aws-cdk/aws-sqs';
```

Comment:
Changed imports to: 
```
import * as cdk from '@aws-cdk/core';
import * as sqs from '@aws-cdk/aws-sqs';
import * as iot from '@aws-cdk/aws-iot';
import * as kms from '@aws-cdk/aws-kms';
import * as iam from '@aws-cdk/aws-iam';
import * as defaults from '@aws-solutions-constructs/core';
```

Used `defaults` for `@aws-solutions-constructs/core` because I saw that used other places in the project.

Reply:
Updated the architecture diagram. Used the SQS service icon instead of the Queue icon.